PWGHF/HFC/Tasks/taskCorrelationHfeHadrons.cxx

```diff
@@ -0,0 +1,287 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file taskCorrelationHfeHadrons.cxx
+/// \author Rashi gupta <rashi.gupta@cern.ch>, IIT Indore
+/// \author Ravindra Singh <ravindra.singh@cern.ch>, IIT Indore
+
+#include "Common/Core/PID/TPCPIDResponse.h"
+#include "Common/Core/TrackSelection.h"
+#include "Common/DataModel/Centrality.h"
+#include "Common/DataModel/EventSelection.h"
+#include "Common/DataModel/Multiplicity.h"
+#include "Common/DataModel/PIDResponse.h"
+#include "Common/DataModel/TrackSelectionTables.h"
+#include "DataFormatsEMCAL/AnalysisCluster.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/HistogramRegistry.h"
+#include "Framework/runDataProcessing.h"
+#include "PWGJE/DataModel/EMCALClusters.h"
+
+#include "Common/Core/RecoDecay.h"
+#include "THnSparse.h"
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+using namespace o2::soa;
+
+using GTrks = o2::soa::Join<o2::aod::Tracks, o2::aod::TracksExtra, o2::aod::pidTPCFullEl, o2::aod::TracksDCA, o2::aod::TrackSelection, o2::aod::TrackSelectionExtension>;
+using GTrk = GTrks::iterator;
+
+struct HfTaskCorrelationHfeHadrons {
+
+  Configurable<bool> isrun3{"isrun3", true, "Data is from Run3 or Run2"};
+  Configurable<bool> clusterinfo{"clusterinfo", true, "EMCal cluster info before and after track match"};
```

Comment:
```suggestion
  Configurable<bool> isRun3{"isRun3", true, "Data is from Run3 or Run2"};
  Configurable<bool> clusterInfo{"clusterInfo", true, "EMCal cluster info before and after track match"};
```
Please pay attention to the proper capitalisation.
See and fix other names in the rest of the code as well.